src/main/java/com/redhat/devtools/lsp4ij/features/workspaceSymbol/LSPWorkspaceRequestedSymbolTypes.java

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.redhat.devtools.lsp4ij.features.workspaceSymbol;
+
+/**
+ * The types of workspace symbols being requested in {@link LSPWorkspaceSymbolParams}.
+ */
+enum LSPWorkspaceRequestedSymbolTypes {
```

Comment:
This is the symbolic discriminator that's used to convey the originating action to the actual LSP engagement.